launch.go

```diff
@@ -653,6 +665,18 @@ func main() {
 			Name:  "cache-max-size-mb",
 			Usage: "Cache allowed max size in mb",
 		},
+		cli.BoolFlag{
+			Name:  "local-mode",
+			Usage: "Enable local mode",
+		},
+		cli.StringFlag{
+			Name:  "local-build-json",
+			Usage: "Build information for local mode",
+		},
```

Comment:
Pass JSON in the same format as payload from `/v4/builds/{id}`

Reply:
Do we need to copy this comment to a code comment?

Reply:
Add comment